Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:trusty
 MAINTAINER Shaun Jackman <sjackman@gmail.com>
 
 RUN apt-get update \
-	&& apt-get install -y curl g++ git make ruby2.0 ruby2.0-dev uuid-runtime \
+	&& apt-get install -y curl g++ git make python-setuptools ruby2.0 ruby2.0-dev uuid-runtime \
```

Comment:
Yes, the exclusion is deliberate. It's not just `python-setuptools` but `python` that's not installed. If a formula requires Python, I believe it should `depends_on :python unless OS.mac?`